### ISSUE TITLE
[bug] remore orderer abnormal energy consuming

### ISSUE THREAD

**Issue by NoX-programer** [RS485/LogisticsPipes#158]
AartBluestoke, I've not found the way to re-open an issue https://github.com/RS485/LogisticsPipes-Dev/issues/154. So, created new one.
Using dev v.656, on the same height with remote orderer pipe energy consuming seems to be normal but when I trying to open remote orderer on height of pipe +5 / -5 it refused to open and if height difference is 15 blocks then numbers of energy required is incredible crazy.
AartBluestoke, please test it carefully over different distances. I have real need to use it up to 500000 blocks far from remote orderer pipe.
If you are thinking my need is differs from normal you may to keep energy costs (just fix these crazy height ratios) but please add some config option to disable (or fix ratios of) energy costs for just remote orderer.

**Comment by davboecki** [comment]
could you please retest using dev build 658. That should fix the calculation.

**Comment by NoX-programer** [comment]
Well, fine - now, on 658, it is consuming energy equal to distance exactly.
But I found another bug: in my configuration (see image) if (unpowered!) power junction (J) has enough energy for operation, remote orderer pipe (R) is NOT consuming energy from junction at all regardless of without link to that junction I have not enough power for remote ordering. But energy is spend normally if I removing ME interface (I) powering this logistics segment through basic pipe (B).
I tested at ~400000 blocks away from this base.

Well... maybe I have to create another issue for this... Anyway, no matter.
![2013-08-11_22 59 53](https://f.cloud.github.com/assets/4730256/944239/a06d76e0-029f-11e3-9d71-19f7b2b496dc.png)
This happens even if territory is loaded by ChickenChunks chunkloader.
